apps/system/js/updatable.js

```diff
@@ -19,6 +21,49 @@
  * - cancelDownload() to cancel it
  */
 
+// Wrapper to read a settings value
+var setting = (function() {
+  var values = {};
+  return key => {
+
+    // Cache the promise straight away so multiple callers dont
+    // create multiple listeners
```

Comment:
This comment would make more sense if it was moved to where we set the key below.